css/style.css

```diff
@@ -507,33 +505,149 @@ address {
     margin-top: 0em;
 }
 
+.floe .idrc-logo-container svg,
+.floe .hewlett-logo-container svg {
+    padding: 0.5em;
+    width: 100%;
+    height: auto;
+}
+
+/* IE-only hacks for better SVG sizing */
+
+/* Large, two column */
+@media all and (min-width: 64em) and (-ms-high-contrast:none)
```

Comment:
@waharnum a potential issue with this approach is that if the user has made use of contrast themes in Windows; which is a possibility for a user who may be setting contrast preferences in UIO. In that case, the height won't be applied properly.

https://msdn.microsoft.com/en-us/library/hh771830(v=vs.85).aspx

Reply:
@waharnum just bumping the comment about using both `active` and `none` in the media query to ensure that the height is applied when a contrast theme is in use on Windows.